### PR TITLE
Windows regression tests

### PR DESCRIPTION
**Pull request by robertmin1**
Windows regression tests

### CODE REVIEW

**Comment by robertmin1** [comment]
Faced some commit issues, was ACK #362

File: .cirrus.yml [hunk 1]
```diff
@@ -152,6 +152,20 @@ task:
...
+    - env:
+        ELECTRUM_MODE: "GUI"
+        QT_QPA_PLATFORM: "offscreen"
+        Binaries: "WINE" 
+      wine_script:
+        - dpkg --add-architecture i386
+        - apt-get update
+        - apt-get -y install  wine-development wine32-development
+        - VERSION=$(git describe --tags --dirty --always | cut -c 1-9)
+        - curl -o namecoin.exe https://api.cirrus-ci.com/v1/artifact/build/$CIRRUS_BUILD_ID/Windows%20build/binaries/contrib/build-wine/dist/electrum-nmc-$VERSION.exe
```
Comment (JeremyRand) [inline .cirrus.yml:164]: Is there a reason why this is named `namecoin.exe`? I think `electrum-nmc.exe` might me more intuitive.

Comment (JeremyRand) [reply .cirrus.yml:164]: (`namecoin.exe` sounds more like a Namecoin Core thing.)

Comment (robertmin1) [reply .cirrus.yml]: Alright!

**Comment by JeremyRand** [comment]
ACK f8e00d90580814ed0d73276cbf0872ec822173c6

**Comment by JeremyRand** [comment]
Manually merged as https://github.com/namecoin/electrum-nmc/commit/f04978b684518c33751bd851243c128b76a463b1. Let me know if I need to do anything else on this one.

**Comment by robertmin1** [comment]
Looks good!

**Pull request closed without merging**